### ISSUE TITLE
Error running test.py with Python 3.6

### ISSUE THREAD

**Issue by rjollos** [stsievert/python-drawnow#7]
Any idea of what the problem might be here? Could it be specific to Python 3.6?

```
$python -m test.py
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "/Users/rjollos/Documents/Workspace/otonexus/python-drawnow/tests/test.py", line 40, in <module>
    x_hat = approx(x, k)
  File "/Users/rjollos/Documents/Workspace/otonexus/python-drawnow/tests/test.py", line 11, in approx
    sigmas[k:] = 0
TypeError: slice indices must be integers or None or have an __index__ method
```

**Comment by rjollos** [comment]
Looks like issue is that `k_values` are `float64`.  So a fix along the lines of:

```diff
diff --git a/tests/test.py b/tests/test.py
index d94e1ed..4ca050d 100644
--- a/tests/test.py
+++ b/tests/test.py
@@ -37,5 +37,5 @@ def draw_fig():
 x = imread('mandrill.png')
 k_values = around(logspace(0.1, log10(64), num=10))
 for k in k_values:
-    x_hat = approx(x, k)
+    x_hat = approx(x, int(k))
     drawnow(draw_fig)
```

However, I get another traceback after that change:
```
Traceback (most recent call last):
  File "/Users/rjollos/Documents/Workspace/otonexus/venv/lib/python3.6/site-packages/matplotlib/backends/backend_macosx.py", line 83, in _draw
    self.figure.draw(renderer)
  File "/Users/rjollos/Documents/Workspace/otonexus/venv/lib/python3.6/site-packages/matplotlib/artist.py", line 55, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "/Users/rjollos/Documents/Workspace/otonexus/venv/lib/python3.6/site-packages/matplotlib/figure.py", line 1299, in draw
    renderer, self, artists, self.suppressComposite)
  File "/Users/rjollos/Documents/Workspace/otonexus/venv/lib/python3.6/site-packages/matplotlib/image.py", line 138, in _draw_list_compositing_images
    a.draw(renderer)
  File "/Users/rjollos/Documents/Workspace/otonexus/venv/lib/python3.6/site-packages/matplotlib/artist.py", line 55, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "/Users/rjollos/Documents/Workspace/otonexus/venv/lib/python3.6/site-packages/matplotlib/axes/_base.py", line 2437, in draw
    mimage._draw_list_compositing_images(renderer, self, artists)
  File "/Users/rjollos/Documents/Workspace/otonexus/venv/lib/python3.6/site-packages/matplotlib/image.py", line 138, in _draw_list_compositing_images
    a.draw(renderer)
  File "/Users/rjollos/Documents/Workspace/otonexus/venv/lib/python3.6/site-packages/matplotlib/artist.py", line 55, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "/Users/rjollos/Documents/Workspace/otonexus/venv/lib/python3.6/site-packages/matplotlib/image.py", line 566, in draw
    renderer, renderer.get_image_magnification())
  File "/Users/rjollos/Documents/Workspace/otonexus/venv/lib/python3.6/site-packages/matplotlib/image.py", line 793, in make_image
    unsampled=unsampled)
  File "/Users/rjollos/Documents/Workspace/otonexus/venv/lib/python3.6/site-packages/matplotlib/image.py", line 484, in _make_image
    output = self.to_rgba(output, bytes=True, norm=False)
  File "/Users/rjollos/Documents/Workspace/otonexus/venv/lib/python3.6/site-packages/matplotlib/cm.py", line 257, in to_rgba
    raise ValueError("Floating point image RGB values "
ValueError: Floating point image RGB values must be in the 0..1 range.
/Users/rjollos/Documents/Workspace/otonexus/venv/bin/python3: Error while finding module specification for 'test.py' (AttributeError: module 'test' has no attribute '__path__')
```

**Comment by stsievert** [comment]
Closed by e998e916ac5b6625023b97cefac6b25f14984fbf and version bumped on PyPI
